examples/urlInterpolationDemo.html

```diff
@@ -41,6 +41,7 @@ <h2>Localization Demo</h2>
             (function () {
                 var DEFAULT_LANG_TAG = 'en-us';
                 var langTag = null;
+                var conversationId = null;
```

Comment:
nit: this conversationId var is not needed at this scope as it's unused in the rest of the function.